en/developer_guide/media_standalone_display.rst

```diff
@@ -0,0 +1,184 @@
+
+It is still possible to add ``MonologBundle`` for the logging.
+
+Installation with the database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the case that you will require to request the database to extract more
+information about the ``media`` object for instance. You will need to create two
+documents : ``EmbedKeyword`` and ``TranslatedValue``.
+
+The ``EmbedKeyword`` document :
+
+.. code-block:: php
+
+    use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+    use OpenOrchestra\ModelInterface\Model\EmbedKeywordInterface;
+    use OpenOrchestra\ModelInterface\Model\KeywordInterface;
+
+    /**
+     * @ODM\EmbeddedDocument
+     */
+    class EmbedKeyword implements EmbedKeywordInterface, KeywordInterface
+    {
+        public static function createFromKeyword(KeywordInterface $keyword)
+        {
+            trigger_error('this method should not be used in the demo');
+
+            $keyword = new EmbedKeyword();
+            $keyword->setLabel($keyword->getLabel());
+
+            return $keyword;
+        }
+
+        /**
+         * @ODM\Id()
+         */
+        protected $id;
+
+        /**
+         * @ODM\Field(type="string")
+         */
+        protected $label;
+
+        public function getId()
+        {
+            return $this->id;
+        }
+
+        public function getLabel()
+        {
+            return $this->label;
+        }
+
+        public function setLabel($label)
+        {
+            $this->label = $label;
+        }
+    }
+
+The ``TranslatedValue`` document :
+
+.. code-block:: php
+
+    use Doctrine\ODM\MongoDB\Mapping\Annotations as ODM;
+    use OpenOrchestra\ModelInterface\Model\TranslatedValueInterface;
+
+    /**
+     * @ODM\EmbeddedDocument
+     */
+    class TranslatedValue implements TranslatedValueInterface
+    {
+        /**
+         * @ODM\Field(type="string")
+         */
+        protected $language;
+
+        /**
+         * @ODM\Field(type="string")
+         */
+        protected $value;
+
+        public function setLanguage($language)
+        {
+            $this->language = $language;
+        }
+
+        public function getLanguage()
+        {
+            return $this->language;
+        }
+
+        public function setValue($value)
+        {
+            $this->value = $value;
+        }
+
+        public function getValue()
+        {
+            return $this->value;
+        }
+    }
+
+Once the document has been created inside your bundle, you should configure the interface
+resolution in the ``config.yml`` file:
+
+.. code-block:: yaml
+
+    doctrine_mongodb:
+        resolve_target_documents:
+                OpenOrchestra\ModelInterface\Model\TranslatedValueInterface: OpenOrchestra\MediaDemoBundle\Document\TranslatedValue
+                OpenOrchestra\ModelInterface\Model\EmbedKeywordInterface: OpenOrchestra\MediaDemoBundle\Document\EmbedKeyword
+
+You have now the opportunity to query the database to get any of the media stored inside.
```

Comment:
I don't understand what is the purpose of these two documents. Shouldn't you give an explanation of what it lets you do or an example of usage ?